front/ui/ui-charts/src/spaceTimeChart/lib/types.ts

```diff
@@ -119,6 +119,12 @@ export type DrawingFunctionHandler = (
     | { type: 'rendering'; layer: LayerType; fn: DrawingFunction }
 ) => void;
 
+export type DrawSinglePointFunction = (
+  ctx: CanvasRenderingContext2D,
+  stcContext: SpaceTimeChartContextType,
+  point: DataPoint
+) => void;
```

Comment:
Nit: do we really need to define a new type here? Can't we just specify types for the function arguments and return value?